typescript/package.json

```diff
@@ -22,10 +22,11 @@
         "validate": "npm run typecheck && npm run lint",
         "storybook": "storybook dev -p 6006",
         "build-storybook": "storybook build",
-        "storybook:test": "test-storybook",
-        "docker:build": "docker build --tag wsc_story_test - < Dockerfile.test",
-        "docker:storybook:test": "docker run -it --rm --mount type=bind,source=.,target=/test --network host --ipc=host wsc_story_test",
-        "docker:storybook:test:update": "npm run docker:storybook:test -- npm run storybook:test -- --updateSnapshot",
+        "storybook:test": "test-storybook --no-index-json",
+        "docker:build": "docker build --tag wsc_story_test --file Dockerfile.test .",
+        "docker:storybook:test": "cross-env-shell CWD=$PWD npm run docker:storybook:test:docker:core -- -it --rm --mount type=bind,source=${CWD}/packages,target=/test/packages --network host --ipc=host wsc_story_test",
+        "docker:storybook:test:update": "npm run docker:storybook:test -- npm run storybook:test -- --url http://host.docker.internal:6006/ --no-index-json --updateSnapshot",
```

Comment:
Shouldn't we use non default port number, to avoid the risk to interfere with other running storybooks ?

Reply:
We could run both the storybook and the tests in the docker container. So you wouldn't have to manually run the storybook at all.

We could do something like
1. Build storybook static
2. Copy into docker image when the image is built
3. In the container, serve storybook and test in parallel as in the [typescript workflow](https://github.com/equinor/webviz-subsurface-components/blob/258638ab5a5d56c4a0ed9c4595417f0be31cd7f5/.github/workflows/typescript.yml#L143)

Should we follow this up in a separate PR?

Reply:
I'm in favor to have that in a separate PR.
The current PR bring snapshot updates to windows users, which is important.